README.md

```diff
@@ -19,38 +19,40 @@ To build Shotover from source please refer to [the contributing documentation](h
 
 Shotover is a high performance, configurable and extensible L7 data-layer proxy for controlling, managing and modifying the flow of database requests in transit. It can be used to solve many different operational and interoperability challenges by transparently intercepting and transforming queries. It is transparent in the sense that it can be plugged into your architecture without requiring application change.
 
-Shotover currently supports intercepting requests for the following technologies (sources):
+Shotover currently supports the following databases:
```

Comment:
should we choose another word? since kafka is not a database

Reply:
good point, reverted to "technologies"